src/library_gl.js

```diff
@@ -921,15 +921,21 @@ var LibraryGL = {
         ret = allocate(intArrayFromString(GLctx.getParameter(name_)), 'i8', ALLOC_NORMAL);
         break;
       case 0x1F02 /* GL_VERSION */:
-        var glVersion = GLctx.getParameter(GLctx.VERSION);
-        // return GLES version string corresponding to the version of the WebGL context
-#if USE_WEBGL2
-        if (GLctx.canvas.GLctxObject.version >= 2) glVersion = 'OpenGL ES 3.0 (' + glVersion + ')';
-        else
+        var glVersion;
+#if WEBGL2_BACKWARDS_COMPATIBILITY_EMULATION
+        glVersion = 1;
+#else
+        glVersion = GLctx.canvas.GLctxObject.version;
 #endif
-        {
-          glVersion = 'OpenGL ES 2.0 (' + glVersion + ')';
+        if (glVersion == 2) {
```

Comment:
should this be `>=` to match how it was before?

Reply:
I can't quite understand why `>=` was used, which seems to be a N:1 mapping. IIUC, `GLctx.canvas.GLctxObject.version` refers to the major version of WebGL, and it's 1:1 mapped to OpenGL ES.

Reply:
Yeah, not sure either, but maybe it was to handle WebGL versions after 2. Which may be irrelevant anyhow given WebGPU is now the focus.

Anyhow, I'm ok with leaving it as `==`, but maybe @juj will have thoughts on this.